layout/PrimaryNav/PrimaryNavLinkSeperator/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+import React from 'react'
+
+export default React.createClass({
+  displayName: 'PrimaryNavLinkSeperator',
+
+  render: function() {
+    return (
+      <hr className="Navigation_linkSeperator" />
+    );
+  }
+});
```

Comment:
This component might be cool to see as a functional component since it keeps no state, e.g.

```js
import React from 'react'

export default () => {
  return (
    <hr className="Navigation_linkSeperator" />
  )
}
```

Up to you, but this is a nice way to reduce a little boilerplate